frontend/src/game/scenes/Collect.ts

```diff
@@ -90,7 +110,26 @@ export class Collect extends Scene {
 
     for (let bit of this.clickyBits) {
       if (bit.toDestroy) {
-        bit.destroyObj();
+        bit.destroy();
+      }
+    }
+
+    for (const bit of this.bitGroup.getChildren()) {
+      for (const virus of this.virusGroup.getChildren()) {
+        if (this.physics.overlap(bit, virus)) {
+          let xPosition = bit.body.position.x;
+          let yPosition = bit.body.position.y;
+          bit.destroy();
+          this.createVirus(xPosition, yPosition);
+          //these variables are passed in because the poistion itself was giving an error, but if I pass it to a varaible it doesn't
+        }
+      }
+    }
+    for (const currentVirus of this.virusGroup.getChildren()) {
```

Comment:
Phaser doesn't allow you to set a collider on an object once? You have to check for an overlap every single frame and then add it manually? Ew.

Reply:
Honestly you probably can, I think we did this as a hack to avoid using the physics simulator at the time.

Reply:
You don't actually and I just figured out how to fix this so gimme a minute and I'll resend.